lms/djangoapps/learner_home/views.py

```diff
@@ -665,6 +656,17 @@ def get(self, request):
         if not recommended_courses:
             return general_recommendations_response
 
+        # Emits an event to track student dashboard page visits.
+        segment.track(
+            user_id,
+            "edx.bi.user.recommendations.viewed",
+            {
+                "is_personalized_recommendation": not is_control,
+                "is_control": is_control if has_is_control else None,
+                "course_keys": [course["course_key"] for course in recommended_courses],
```

Comment:
I had a chat with James from DE team and he suggested that we rename the `course_key` property.

```suggestion
                "course_key_array": [course["course_key"] for course in recommended_courses],
```